src/Coordination/KeeperDispatcher.cpp

```diff
@@ -652,10 +652,11 @@ uint64_t KeeperDispatcher::getSnapDirSize() const
 Keeper4LWInfo KeeperDispatcher::getKeeper4LWInfo() const
 {
     Keeper4LWInfo result;
-    result.is_follower = server->isFollower();
+    result.is_follower = isFollower();
```

Comment:
We need to store the value of `isLeader` in a variable and not call the function twice.

Reply:
Agree